v2/apiserver/internal/core/projects.go

```diff
@@ -200,6 +206,71 @@ func (p *projectsService) Create(
 			errors.Wrapf(err, "error storing new project %q", project.ID)
 	}
 
+	// Make the current user an admin, developer, and user of the project
+	principal := libAuthn.PrincipalFromContext(ctx)
+
+	var principalRef authz.PrincipalReference
+	switch prin := principal.(type) {
+	case *authn.User:
+		principalRef = authz.PrincipalReference{
+			Type: authz.PrincipalTypeUser,
+			ID:   prin.ID,
+		}
+	case *authn.ServiceAccount:
+		principalRef = authz.PrincipalReference{
+			Type: authz.PrincipalTypeServiceAccount,
+			ID:   prin.ID,
+		}
+	default:
+		return project, nil
```

Comment:
So in this case do we indeed have a project that is "useless", i.e. no principals are authz'd to interact w/ it?

Reply:
The only plausible and likely scenario here is that the principal is the root user. The root user already has the admin bit on all projects (scope `*`).